src/scenes/TradeHub/index.js

```diff
@@ -3,49 +3,42 @@ import './styles.css';
 
 // Libraries
 import binance from 'binance-api';
+import { Container, Row, Col } from 'reactstrap';
 
 // Components
 import Header from '../../components/Header';
 import HorizontalTabList from '../../components/HorizontalTabList';
 import InfoPanel from '../../components/InfoPanel';
+import InputPanel from '../../components/InputPanel';
+import OptionPanel from '../../components/OptionPanel';
 import Chart from '../../components/Chart';
 
-class TradeHub extends Component {
-  constructor(props) {
-    super(props);
-
-    this.state = {
-      cryptoList: [],
-      selectedCrypto: {symbol:"ETHBTC", baseAsset:"ETH", quoteAsset:"BTC"},
-      currentPrice: 0,
-      boughtPrice: 0,
-      diffPercentage: 0.01,
-      highestPrice: 0,
-      messages: [],
-      sellEnabled: false,
-      hasSold: false,
-      socketKeys: ["ETHBTC@kline_1m"]
-    }
-
+export default class TradeHub extends Component {
+  state = {
+    cryptoList: [],
+    selectedCrypto: { symbol: 'ETHBTC', baseAsset: 'ETH', quoteAsset: 'BTC' },
+    currentPrice: 0,
+    boughtPrice: 0,
+    diffPercentage: 0.01,
+    highestPrice: 0,
+    messages: [],
+    sellEnabled: false,
+    hasSold: false,
+    socketKeys: ['ETHBTC@kline_1m']
+  };
+
+  componentDidMount = () => {
     binance.options({
-      'APIKEY':      props.opts.binance.key,
-      'APISECRET':   props.opts.binance.secret,
-      'reconnect': false
+      APIKEY:      this.props.opts.binance.key,
+      APISECRET:   this.props.opts.binance.secret,
+      reconnect: false
     });
 
-    // bindings
-    this.setBoughtPrice = this.setBoughtPrice.bind(this);
-    this.setDiffPercentage = this.setDiffPercentage.bind(this);
-    this.toggleSellEnabled = this.toggleSellEnabled.bind(this);
-    this.changeSelectedCrypto = this.changeSelectedCrypto.bind(this);
-  }
-
-  componentDidMount() {
     this.getCryptoList();
     this.bindSocket(this.state.selectedCrypto.symbol);
-  }
+  };
 
-  rebindSocket() {
+  rebindSocket = () => {
     const newCrypto = this.state.selectedCrypto.symbol;
     const newEndpoint = newCrypto.toLowerCase() + "@kline_1m";
```

Comment:
Use single quotes

Reply:
We should update the code guidelines, to document that we only use single quotes.